documentation/static/servers.json

```diff
@@ -117,6 +117,23 @@
     "endorsed": true,
     "environmentVariables": []
   },
+  {
+    "id": "apify-mcp-server",
+    "name": "Apify",
+    "description": "Access thousands of ready-made web scraping and automation Actors from the Apify platform.",
```

Comment:
Pls let's change this to: 
> Extract data from any website with thousands of scrapers, crawlers, and automations on Apify Store.

Reply:
It will display and be used for search on https://block.github.io/goose/extensions/